chain/client/src/client_actor.rs

```diff
@@ -1671,6 +1672,29 @@ impl ClientActor {
 
                     let use_colour =
                         matches!(self.client.config.log_summary_style, LogSummaryStyle::Colored);
+
+                    // Notify each shard to sync.
+                    let shard_layout = self
+                        .client
+                        .epoch_manager
+                        .get_shard_layout(&epoch_id)
+                        .expect("Cannot get shard layout");
+
+                    let sync_hash = state_sync_status.sync_hash;
+                    for &shard_id in &shards_to_sync {
+                        let shard_uid = ShardUId::from_shard_id_and_layout(shard_id, &shard_layout);
+                        match self.client.state_sync_adapter.clone().read() {
+                            Ok(sync_adapter) => sync_adapter.send(
+                                shard_uid,
+                                (SyncMessage::StartSync(SyncShardInfo { shard_uid, sync_hash }))
```

Comment:
Same here. Do you start sync every few milliseconds that this `run_sync_step()` function is triggered?

Reply:
That was unnecessarily too much indeed. I changed it to only be called once.